features/base/styles/components/web/shimStyles.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * Shim style properties to work correctly on Web.
```

Comment:
Can you please add better description on what is "shim style" and why do we need it.